packages/core/src/styles/typography/_typography.scss

```diff
@@ -182,13 +182,18 @@
 }
 
 [cds-text~='link'] {
-  color: $cds-alias-status-info !important;
+  color: $cds-global-typography-link-color !important;
   text-decoration: underline !important;
   line-height: inherit !important;
   font-size: inherit !important;
 
   &:hover {
-    color: $cds-alias-status-info-shade !important;
+    filter: brightness(85%);
+  }
+
+  &:visited:not([cds-text~='static']),
+  &[cds-text~='visited']:not([cds-text~='static']) {
```

Comment:
This would be considered a new API, `static` disabling the visited color and `visited` for forcing the visited color regardless if the link was clicked. This is useful for applications where the navlinks dont need visited state or where the visited state needs to be controlled by the host router.

Reply:
Do we need to add documentation about that to the website?

Reply:
Yeah eventually it should. It falls under typography/foundation for core which is not yet in the website. The website points to storybook right now until those are added. I did update the storybook demo docs to show examples of each https://clarity.design/foundation/typography/

Reply:
Also added note open to different names, I wasn't sure what to call the disabled visited style option. Maybe `no-visit` would be more clear?

Reply:
Is that a link that has been visited by the user but currently disabled?

Reply:
Its for the use case of styling the link but don't change the color if it has been visited. Example if there are links being used in some kind of navigation sometimes apps don't want to show the visited purple color since eventually all the links would be visited.

Reply:
Gotcha. I think `static` is ok there; `no-visit` seems more ambiguous. Also open to different / clearer names but don't have any suggestions atm. :)